pkg/operator/helpers/chart/render.go

```diff
@@ -41,33 +41,33 @@ func NewDefaultKlusterletChartConfig() *KlusterletChartConfig {
 	}
 }
 
-func RenderClusterManagerChart(config *ClusterManagerChartConfig, namespace string) ([][]byte, error) {
+func RenderClusterManagerChart(config *ClusterManagerChartConfig, namespace string) ([][]byte, [][]byte, error) {
```

Comment:
better to add a comment with output description

Reply:
fixed, ptal